app/views/batch_imports/_form_metadata.html.erb

```diff
@@ -9,11 +9,11 @@
         input_html: { class: 'form-control' } %>
 
     <div class="with-headroom" style="padding-bottom:25px;">
-        <label>CSV Metadata Source</label><br />
```

Comment:
Could the things like labels and etc.  extracted into our internationalization file for the application? It would be nice to follow that pattern for everything, which follows the community and take advantage of the work in Blacklight and Hyrax in this context.